src/applications/yellow-ribbon/components/SearchResult/index.jsx

```diff
@@ -129,13 +129,15 @@ export const SearchResult = ({ school }) => (
             Maximum Yellow Ribbon funding amount
             <br />
             (per student, per year)
+            <span className="sr-only">:</span>
```

Comment:
Can you give this a listen in Safari+VO locally? I had to insert the colon before the data value in order to get it to read out clearly.

Reply:
I just listened to it on safari and it went like this:
```
Funding available for
colon
All eligible students
```
It seemed easy to understand imo 🙂

Reply:
Great work! Once it's in staging, if you'd like me to give it a quick run with an axe scan and a screen reader, lmk. Otherwise, we can wait for staging review. 

Thanks for the quick work!